### DISCUSSION TITLE
Get qualifying results and race results

### DISCUSSION THREAD

**Discussion by JGeek00** [br-g/openf1#5] [Q&A] [answered]
How can I get the qualifying results and the race results of a certain event (example: qualifying results and race results of the Bahrain GP)?

**Comment by br-g** [comment, marked as answer]
Hello, I got this request a lot and it has been added to the early 2024 roadmap!

**Reply by krumpyzoid** [reply]
Will definitely be appreciated. If possible separate `Qualifying` into `Q1`, `Q2` and `Q3` sessions would also make it easier to work with qualifying data

**Comment by fabdeceglia** [comment]
any ETA on this functionality?
